CONTRIBUTING.md

```diff
@@ -7,6 +7,7 @@ You can take part in making Shopsys Framework better.
 * [Guidelines for Creating Commits](./docs/contributing/guidelines-for-creating-commits.md)
 * [Guidelines for Writing Documentation](./docs/contributing/guidelines-for-writing-documentation.md)
 * [Guidelines for Pull Request](./docs/contributing/quidelines-for-pull-request.md)
+* [Guidelines for Dependencies](./docs/contributing/quidelines-for-dependencies.md)
```

Comment:
There is a typo in the file name: `quidelines` instead of `guidelines`.

Could you please fix it in the article about pull requests as well?